pkc

```diff
@@ -310,6 +310,19 @@ while [[ $# -gt 0 ]]; do
             ./aqua/MediaWiki_Backup/restore_all.sh "$most_recent"
             break
             ;;
+        nuke)
+            read -r -p $'This will destroy your containers and delete backup data. \n\tAre you sure? [y/N] ' response
+            case "$response" in
+                [yY][eE][sS]|[yY]) 
+                    sudo docker-compose down && rm -rf mountPoint
```

Comment:
It usually requires `sudo rm -rf` on my laptop.

Reply:
Also, to be safer, the nuke command should detect that we are in the right folder, i.e. docker-compose.yml exists (so that `docker-compose down` works), and that it is a micro-PKC repo.

Reply:
I tried: `sudo rm -r` without `-f` also works. We should not use the `-f` flag.